### DISCUSSION TITLE
Help needed with Resend Medusa config

### DISCUSSION THREAD

**Discussion by jarmovdh** [pevey/medusa-plugin-resend#4] [Q&A]
When i run my backend server i receive the following error:

when adding this object to the medusa-config-js

```
warn:    The subscriber in /node_modules/medusa-plugin-resend/subscribers/order.js is missing a config.
warn:    The subscriber in /node_modules/medusa-plugin-resend/subscribers/restock.js is missing a config.
warn:    The subscriber in /backend/node_modules/medusa-plugin-resend/subscribers/user.js is missing a config.
error:   Error starting server
TypeError: Cannot read properties of undefined (reading 'startsWith')
    at new ResendService (/Users/jarmovanderheul/Repos/411/411-backend/node_modules/medusa-plugin-resend/services/resend.js:66:59)

const plugins = [


  },
  {
    resolve: "medusa-plugin-resend",
       options: {
          api_key: process.env.RESEND_API_ID,
          from: process.env.RESEND_FROM,
       }
    },
]
```

**Comment by Hermitauge** [comment]
you gotta provide the rest of the options, i dont have the full but it goes something like this 
      from: process.env.RESEND_FROM,
      template_path: process.env.RESEND_TEMPLATE_PATH,
      subject_template_type: process.env.RESEND_SUBJECT_TEMPLATE_TYPE,
      body_template_type: process.env.RESEND_BODY_TEMPLATE_TYPE,
      ...and the templates i think they are something like:
      RESEND_USER_PASSWORD_RESET=user_password_reset
RESEND_USER_INVITE=user_invite

make sure you have the templates folder with handbars in data/templates for each resend email type
